src/coreclr/vm/callhelpers.cpp

```diff
@@ -468,11 +468,32 @@ void MethodDescCallSite::CallTargetWorker(const ARG_SLOT *pArguments, ARG_SLOT *
 
                 switch (stackSize)
                 {
+#if defined(TARGET_LOONGARCH64)
+                    case 1:
+                        if (m_argIt.GetArgType() == ELEMENT_TYPE_U1)
```

Comment:
`ELEMENT_TYPE_BOOLEAN` and `ELEMENT_TYPE_CHAR` should be treated as unsigned. They are unsigned types in .NET.

Reply:
Thanks.

The `ELEMENT_TYPE_CHAR`  is also unsigned ?

Reply:
Yes, `ELEMENT_TYPE_CHAR` is unsigned.

Reply:
Thanks, I will update it.